CODEOWNERS

```diff
@@ -42,3 +51,22 @@ packages/shared_preferences/shared_preferences_ios/**    @cyanglaz
 packages/url_launcher/url_launcher_ios/**                @jmagman
 packages/video_player/video_player_avfoundation/**       @hellohuanlin
 packages/webview_flutter/webview_flutter_wkwebview/**    @cyanglaz
+
+# - Linux
```

Comment:
@cbracken I gave you all of the desktop plugins with the idea that you'll delegate ownership over time; if you have any you want to delegate now, let me know.

(I also expect that in practice I'll still be a primary reviewer for a lot of desktop implementations, but I figured that putting you instead of me will remind me that we're supposed to be working on handing those off incrementally :) )